app/src/test/java/ar/com/wolox/android/example/ui/utils/RetrofitServicesMockManager.kt

```diff
@@ -0,0 +1,79 @@
+package ar.com.wolox.android.example.ui.utils
+
+import ar.com.wolox.android.example.model.User
+import ar.com.wolox.android.example.service.UserAuthService
+import org.mockito.ArgumentMatchers.any
+import org.mockito.Mockito.`when`
+import org.mockito.Mockito.mock
+import retrofit2.Call
+import retrofit2.Callback
+import retrofit2.Response
+import java.util.Collections.singletonList
+
+class RetrofitServicesMockManager {
+
+    companion object {
+
+        private const val correctEmail: String = "email@email.com"
+        private const val correctPassword: String = "password"
+        private const val incorrectPassword: String = "incorrectPassword"
+        private const val incorrectEmail: String = "incorrectEmail@email.com"
+
+        fun getCorrectUser(): User {
```

Comment:
Just to promote Kotlin's tips, I would recommend to redifine every mock function using `apply`, also to see that a function in kotlin can be define using `fun myFunction() = something`
For example:
```
fun getCorrectUser() = User().apply { ... } // without return
```

Reply:
the same for the rest of the functions